pkgs/servers/web-apps/pict-rs/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv
+, lib
+, fetchFromGitea
+, rustPlatform
+, makeWrapper
+, protobuf
+, Security
+, imagemagick
+, ffmpeg
+, exiftool
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pict-rs";
+  version = "0.3.0-alpha.37";
+
+  src = fetchFromGitea {
+    domain = "git.asonix.dog";
+    owner = "asonix";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-21yfsCicn2bjSNEMMWDG8wvnw10uT3M1L3cXCUhc24c=";
+  };
+
+  cargoSha256 = "sha256-F/mqdIjF5QOq5Plnq0DyeFP1+b7dCBcoU9iFxzcaZws=";
+
+  # needed for internal protobuf c wrapper library
+  PROTOC = "${protobuf}/bin/protoc";
+  PROTOC_INCLUDE = "${protobuf}/include";
+
+  buildInputs = [ makeWrapper ] ++ lib.optionals stdenv.isDarwin [ Security ];
```

Comment:
makeWrapper belongs into nativeBuildInputs.

Reply:
ok, I will make a follow up PR for the service and will address those comments there.
Thank you for the review!

Reply:
taken care of in https://github.com/NixOS/nixpkgs/pull/139868 in case you are interested